Palo Alto Networks/paloalto-ngfw/ingest/parser.yml

```diff
@@ -490,6 +604,10 @@ stages:
           rule.uuid: "{{parsed_event.message.PanOSRuleUUID or parsed_event.message.RuleUUID}}"
           source.bytes: "{{parsed_event.message.BytesSent or parsed_event.message.in}}"
 
+      - set:
+          event.timezone: "{{'+' + parsed_event.message.HighResolutionTimestamp.split('+')[1]}}"
+        filter: "{{parsed_event.message.HighResolutionTimestamp != None}}"
+
```

Comment:
`HighResolutionTimestamp` must be used to set `@timestamp`. Can you use it in a date.parse stage? Thanks